src/tests/Common/CLRTest.CrossGen.targets

```diff
@@ -100,6 +100,10 @@ if [ ! -z ${RunCrossGen2+x} ]%3B then
 
         echo $2 >> $__ResponseFile
 
+
```

Comment:
@jkoritzinsky I was wondering if this part of the change looks good to you. We didn't have a way to pass the  `-O` (optimize) option to crossgen and it seemed to me that makes sense to set it based on the test csproj <Optimize> field. So when a test is requested to be built with optimizations, the crossgened would also be optimized.

Reply:
Yeah that makes sense to me as well.

Reply:
crossgen should pickup the optimization setting from the test binary: https://github.com/dotnet/runtime/blob/5c9f64cce935e20603fd101d5d9295e8029021a6/src/coreclr/tools/aot/crossgen2/Program.cs#L557-L560 . I think passing the `-O` explicitly is redundant here.

Reply:
It was not working for some reason. I've added the <Optimize> element to the test project, but until I've added passing that -O option, it was not inlining the pinvoke. I've rebuilt the test after adding the <Optimize> manually (dotnet build -c Checked /path/to/the/test/csproj), so I don't know what was going on. Anyways, I'll try the crossgen test running mode to see if I can hit the failure that way.

Reply:
Building all tests using the src/tests/build.sh generated a dll with the same value of the DebuggableAttribute.